config/locales/views/transactions/it.yml

```
@@ -0,0 +1,350 @@
+      pending: In attesa
+      pending_tooltip: Transazione in attesa — potrebbe cambiare alla registrazione
+      linked_with_provider: Collegata con %{provider}
+      activity_type_tooltip: Tipo di attività di investimento
+      possible_duplicate: Duplicato?
+      potential_duplicate_tooltip: Questa potrebbe essere un duplicato di un'altra transazione
+      review_recommended: Da rivedere
+      review_recommended_tooltip: Grande differenza di importo — si consiglia revisione per verificare se è un duplicato
+      split: Suddivisa
+      split_tooltip: Questa transazione è stata suddivisa in più voci
+      split_child_tooltip: Parte di una transazione suddivisa
+    merge_duplicate:
```

YAMLlint (1.37.1)

[error] duplication of key "merge_duplicate" in mapping

(key-duplicates)